data/mods/Globally Unique Specials/overmap/overmap_special/specials.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Isherwood Farms",
+    "flags": [ "FARM", "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "evac_center",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
```

Comment:
The evac center is, for many new players, their first destination (found via the computer). Is making it globally unique liable to make it _extremely_ far from player spawn?

Reply:
Does it not spawn on every overmap? How much do you need it to spawn? Also, calls for vanillization are not strictly related to this completely option _mod_...so I don't know what clueless new players have to do with anything?

Reply:
Well, this PR may be a good candidate for moving to vanilla :) Performance issues are a healthy concern for mods, as well.

Reply:
Ahhh...no, they've talked it down multiple times, it won't be added to vanilla. It's very important that the very special characters are seen...again and again.

Reply:
> Ahhh...no, they've talked it down multiple times, it won't be added to vanilla. It's very important that the very special characters are seen...again and again.

It won't be added to vanilla... until we are able to enforce spawning within a given radius of the player.

Reply:
I wonder if you could artificially crank up the occurances (e.g 40/100), so it will be generated towards the center (and hopefully spawn) and GLOBALLY_UNIQUE takes care that it will not be generated twice.

Reply:
@Leland [#61257](https://github.com/CleverRaven/Cataclysm-DDA/issues/61257?notification_referrer_id=NT_kwDOAEUNTLI0NDcyNzYxMDI4OjQ1MjUzODg&notifications_query=is%3Aunread#issuecomment-1281131036) is literally the latest of multiple discussions on having this in vanilla, and explains why it can't happen for now (TLDR: things like anthills/fungus/triffids can overrun the unique location even before the player gets there)

Reply:
> It won't be added to vanilla... until we are able to enforce spawning within a given radius of the player. 
Something like forcing Exodii, Hub, Refugee, Artisans and Isherwoods to spawn in the middle 4 OMTs and prevent triffids/fungus/ants/etc from spawning in the middle 16 OMTs?

Reply:
OK...so, if copy-from doesn't have support for overmap specials, is there another elegant way I can do this mod? That will not require pasting the specials wholesale into the mod files?